library/Icingadb/Widget/BaseListItem.php

```diff
@@ -39,6 +39,9 @@ public function __construct($item, BaseItemList $list)
         $this->list = $list;
 
         $this->addAttributes($this->baseAttributes);
+        $this->addAttributes([
+            'data-icinga-multiselect-count-label'   => t('%d Items selected')
+        ]);
```

Comment:
Take a look at the `DetailActions` trait. There, register this attribute in `initializeDetailActions()`. Make sure to also use a callback to avoid registering it if it's not necessary.

Reply:
However, I wanted to add this attribute to the individual selectable list items rather than to the `ul`.

Reply:
To the list items? The text is *about* the list items in the context of the list where they're a part of.